### ISSUE TITLE
squash merging

### ISSUE THREAD

**Issue by tunnell** [JelleAalbers/strax#9]
You might want to setup the default to be squash merging to ensure that the commit log stays clean.  This is in settings on the Github project.  I don't have access.

**Comment by JelleAalbers** [comment]
I don't think you can change the default, just disallow options completely. Preserving the commit history with merge can be valuable in some cases, though in others squashing is best. It depends how people like to work and on what the task is.
